website/index.html

```diff
@@ -72,11 +79,25 @@ <h2>Output</h2>
           });
         </script>
       </div>
-      <button>Test</button>
-      <button>Submit</button>
+      <div class="editorButtons">
+        <button>Test</button>
+        <button id="submit-button">Submit</button>
+        <div class="spacer"></div>
+        <select name="languages" id="language-selector">
+          <option value="julia">Julia</option>
+          <option value="ruby">Ruby</option>
+        </select>
+        <script>
+          const updateLang = () => {
+            monaco.editor.setModelLanguage(window.editor.getModel(), document.getElementById("language-selector").value)
+          }
+          document.getElementById("language-selector").onchange = updateLang
+          updateLang()
```

Comment:
`monaco.editor` is not yet defined when this runs for the first time, this should probably happen when page is fully loaded